app/domain/sac_imports/csv_source.rb

```diff
@@ -121,16 +128,25 @@ def initialize(source_name, source_dir: SOURCE_DIR)
     assert_available_source
   end
 
-  def rows
+  def rows(filter: nil)
     data = []
-    CSV.foreach(path, headers: true, encoding: "bom|utf-8") do |row|
-      data << process_row(row)
+    CSV.foreach(path, headers: true, encoding: "bom|utf-8") do |raw_row|
+      row = process_row(raw_row)
+      next unless filter.blank? || filter_match?(row, filter)
```

Comment:
Ich finde `unless` immer sehr verwirrend:

```suggestion
      next if filter && !filter_match?(row, filter)
```

Reply:
rubocop wollte das so ...